lib/ActivityStreams.js

```diff
@@ -57,14 +59,19 @@ const PLACES_CHANGES_EVENTS = [
   "bookmarkChanged"
 ];
 
+const HOME_PAGE_PREF = "browser.startup.homepage";
+
 function ActivityStreams(options = {}) {
   this.options = Object.assign({}, DEFAULT_OPTIONS, options);
   EventEmitter.decorate(this);
 
+  this._newTabURL = `${this.options.pageURL}#/`;
```

Comment:
I know this is only slightly different from what there was before, however, I think a comment explaning why there's a `#/` at the end of the url for readers unfamiliar with how our content code works